src/components/ReanimatedSwipeable.tsx

```diff
@@ -199,6 +200,26 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
     props: SwipeableProps,
     ref: ForwardedRef<SwipeableMethods>
   ) {
+    const {
+      leftThreshold,
+      rightThreshold,
+      onSwipeableOpenStartDrag,
+      onSwipeableCloseStartDrag,
+      enableTrackpadTwoFingerGesture,
+      enabled,
+      containerStyle,
+      childrenContainerStyle,
+      animationOptions,
+      overshootLeft,
+      overshootRight,
+      onSwipeableWillOpen,
+      onSwipeableWillClose,
+      onSwipeableOpen,
+      onSwipeableClose,
+      testID,
+      ...remainingProps
+    } = useMemo(() => props, [props]);
```

Comment:
Same as above

Reply:
I think we do want to keep this one, as the `ForwardedRef` prop may change without changing `SwipeableProps` prop.
While `spread` doesn't take too much resources, we want to avoid triggering multiple `useCallbacks` further down.

Reply:
Same here